lsp-mode.el

```diff
@@ -664,6 +664,13 @@ If this is set to nil, `eldoc' will show only the symbol information."
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-after-apply-edits-hook nil
```

Comment:
as per conventions hooks that receive arguments should be named `*-functions`. We don't follow it(initially, I didn't know about that convention).

Reply:
Also, it does not seem a good convention IMO, since this hook has nothing related to `functions`